src/index.js

```diff
@@ -0,0 +1 @@
+console.log
```

Comment:
there is some error. because you forgot to delete console log. probably you can do it like this
```js
console.log(test)
```